Week 1 - C/mario-more/mario.c

```diff
@@ -0,0 +1,38 @@
+#include <cs50.h>
+#include <stdio.h>
+
+void printHashtag(int n)
```

Comment:
Name is misleading since it said it prints only 1 hashtag, let's improve it 🙏🏾 
```suggestion
void printHashtags(int n)
```